src/Features/Core/Portable/SolutionCrawler/WorkCoordinator.cs

```diff
@@ -575,7 +575,8 @@ private async Task EnqueueProjectConfigurationChangeWorkItemAsync(ProjectChanges
                     !object.Equals(oldProject.AnalyzerOptions, newProject.AnalyzerOptions) ||
                     !object.Equals(oldProject.DefaultNamespace, newProject.DefaultNamespace) ||
                     !object.Equals(oldProject.OutputFilePath, newProject.OutputFilePath) ||
-                    !object.Equals(oldProject.OutputRefFilePath, newProject.OutputRefFilePath))
+                    !object.Equals(oldProject.OutputRefFilePath, newProject.OutputRefFilePath) ||
+                    !object.Equals(oldProject.State.RunAnalyzers, newProject.State.RunAnalyzers))
```

Comment:
💡 If this is a `bool` property, it would be good to use `==` here to avoid boxing